### ISSUE TITLE
ChromeDriver constantly stucks

### ISSUE THREAD

**Issue by yivo** [openware/peatio#244]

**Comment by yivo** [comment]
Found why.

```
[60.723][DEBUG]: DevTools request: http://localhost:12311/json/version
[60.725][DEBUG]: DevTools request failed
[60.775][DEBUG]: DevTools request: http://localhost:12311/json/version
[60.776][DEBUG]: DevTools request failed
[60.827][DEBUG]: DevTools request: http://localhost:12311/json/version
[60.828][DEBUG]: DevTools request failed
[60.878][DEBUG]: DevTools request: http://localhost:12311/json/version
[60.879][DEBUG]: DevTools request failed
[60.929][DEBUG]: DevTools request: http://localhost:12311/json/version
[60.930][DEBUG]: DevTools request failed
[60.981][DEBUG]: DevTools request: http://localhost:12311/json/version
[60.983][DEBUG]: DevTools request failed
[61.034][DEBUG]: DevTools request: http://localhost:12311/json/version
[61.035][DEBUG]: DevTools request failed
[61.086][DEBUG]: DevTools request: http://localhost:12311/json/version
[61.088][DEBUG]: DevTools request failed
[61.138][DEBUG]: DevTools request: http://localhost:12311/json/version
[61.140][DEBUG]: DevTools request failed
[61.191][DEBUG]: DevTools request: http://localhost:12311/json/version
[61.192][DEBUG]: DevTools request failed
[61.243][DEBUG]: DevTools request: http://localhost:12311/json/version
[61.244][DEBUG]: DevTools request failed
[61.294][DEBUG]: DevTools request: http://localhost:12311/json/version
[61.295][DEBUG]: DevTools request failed
[61.345][DEBUG]: DevTools request: http://localhost:12311/json/version
[61.345][DEBUG]: DevTools request failed
[61.395][DEBUG]: DevTools request: http://localhost:12311/json/version
[61.396][DEBUG]: DevTools request failed
[61.446][DEBUG]: DevTools request: http://localhost:12311/json/version
[61.447][DEBUG]: DevTools request failed
[61.497][DEBUG]: DevTools request: http://localhost:12311/json/version
[61.498][DEBUG]: DevTools request failed
[61.549][DEBUG]: DevTools request: http://localhost:12311/json/version
[61.550][DEBUG]: DevTools request failed
[61.552][INFO]: RESPONSE InitSession unknown error: Chrome failed to start: exited abnormally
[61.552][DEBUG]: Log type 'driver' lost 0 entries on destruction
[61.552][DEBUG]: Log type 'browser' lost 0 entries on destruction
```

**Comment by yivo** [comment]
```
[12.206][INFO]: Waiting for pending navigations...
[12.207][INFO]: Done waiting for pending navigations. Status: unexpected alert open
```

**Comment by yivo** [comment]
Fixed in #247.
